client/enterprise_client_oss.go

```diff
@@ -0,0 +1,13 @@
+// +build !ent
+
+package client
+
+type EnterpriseClient struct{}
```

Comment:
Some docs here would be helpful.  Also, do these types (e.g. `EnterpriseClient` and `EnterpriseAgent`) need to be exported?